Update readme for Twenty Twenty-Two to add comments block update (#7628)
WordPress/wordpress-develop: src/wp-content/themes/twentytwentytwo/readme.txt

Whether you’re building a single-page website, a blog, a business website, or a portfolio, Twenty Twenty-Two will help you create a site that is uniquely yours.

== Comment Block Support ==

If you are using WordPress version 6.1 or newer, you are recommended to update the comment block to the latest version.

> ### Instructions:

Reviewer: ```suggestion
= Instructions =
```

To match the format of the other sub headings (see the changelog).

> 1. Open the Site Editor from your WordPress admin area.

Reviewer: ```suggestion
1. From the WordPress admin, open Appearance > Site Editor.
```

2. Select "Templates" from the menu in the Site Editor sidebar.

> 3. Select the template and open it in the Site Editor.

Reviewer: ```suggestion
3. Select the template (see the list below) and open it in the Site Editor.
```

4. Locate the comments block.
5. If the comments block has a button with the text "Switch to editable mode", activate the button.
6. Save the template.

If the button is not present, then you may have already updated the block, and you do not need to make any changes.

> ### Repeat these instructions for the following templates:

Reviewer: ```suggestion
Follow these instructions for each of the following templates:
```
